crates/router/src/types/domain/payments.rs

```diff
@@ -429,6 +429,62 @@ pub struct CardToken {
     pub card_cvc: Option<Secret<String>>,
 }
 
+#[derive(serde::Deserialize, serde::Serialize, Debug, Clone, ToSchema, Eq, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum BankDebitData {
+    AchBankDebit {
+        billing_details: BankDebitBilling,
+        #[schema(value_type = String, example = "000123456789")]
```

Comment:
we do not need the schema attribute here, you can also remove the `ToSchema` in derive for all the types here in domain